start.sh

```diff
@@ -14,9 +27,20 @@ echo Running python startups
 python manage.py migrate
 python manage.py collectstatic --verbosity 0 --noinput
 
-echo "Starting Gunicorn"
+if [ "$DEBUGPY_ENABLE" = "True" ]; then
+      echo "Starting Gunicorn for DEBUGPY debugging"
+      # Workers need to be set to 1 for DEBUGPY
+      GUNICORN_WORKERS=1
+      GUNICORN_RELOAD="--reload"
+      GUNICORN_TIMEOUT=0
+else
+      echo "Starting Gunicorn for Production"
+      GUNICORN_RELOAD=
```

Comment:
missing value here?

Reply:
No, It shouldn't assign anything. I just followed what is in Myla. It could be have better logic, but I feel that i not high priority.

Reply:
Even though it may work, it _looks_ like a mistake.  Zhen and I both thought it looked odd.  Maybe change it to an empty string instead…

```suggestion
      GUNICORN_RELOAD=''
```

Reply:
OK, we will use the same pattern as the one in MyLA project. Mark this as resolved.